doc/release_notes/v3.0.0.rst

```diff
@@ -135,6 +135,9 @@ Fixes
   :meth:`Dataset.update<pydicom.dataset.Dataset.update>` (:issue:`1816`)
 * Fixed :func:`~pydicom.encaps.encapsulate_extended` not returning the correct
   values for odd-length frames (:issue:`1968`)
+* Fixed a ``KeyError`` when comparing codes with one of the codes having
+``scheme_designator`` set to ``SRT`` but not being included in the ``SRT``
+to ``SCT`` code mapping (:issue:`1994`)
```

Comment:
You may have to add 2 spaces at the start of the continuing lines to render this correctly